src/__screenshot_tests__/callout-screenshot-test.tsx

```diff
@@ -0,0 +1,85 @@
+import {openStoryPage, screen} from '../test-utils';
+
+test('Callout', async () => {
+    await openStoryPage({
+        section: 'Components/Dialogs/Callout',
+        name: 'Callout',
+    });
+
+    const callout = await screen.findByRole('alert');
+
+    const image = await callout.screenshot();
+    expect(image).toMatchImageSnapshot();
+});
+
+test('Callout over inverse', async () => {
+    const page = await openStoryPage({
+        section: 'Components/Dialogs/Callout',
+        name: 'Callout',
+    });
+
+    await page.click(await screen.findByLabelText('Over inverse'));
+
+    const callout = await screen.findByRole('alert');
+
+    const image = await callout.screenshot();
+    expect(image).toMatchImageSnapshot();
+});
+
+test('Callout not closable', async () => {
+    const page = await openStoryPage({
+        section: 'Components/Dialogs/Callout',
+        name: 'Callout',
+    });
+
+    await page.click(await screen.findByLabelText('Is closable'));
+
+    const callout = await screen.findByRole('alert');
+
+    const image = await callout.screenshot();
+    expect(image).toMatchImageSnapshot();
+});
+
+test('Callout with ButtonLink', async () => {
+    const page = await openStoryPage({
+        section: 'Components/Dialogs/Callout',
+        name: 'Callout',
+    });
+
+    await page.select(await screen.findByLabelText('actions'), 'link');
+
+    const callout = await screen.findByRole('alert');
+
+    const image = await callout.screenshot();
+    expect(image).toMatchImageSnapshot();
+});
+
+test('Callout without actions', async () => {
+    const page = await openStoryPage({
+        section: 'Components/Dialogs/Callout',
+        name: 'Callout',
+    });
+
+    await page.select(await screen.findByLabelText('actions'), 'none');
+
+    const callout = await screen.findByRole('alert');
+
+    const image = await callout.screenshot();
+    expect(image).toMatchImageSnapshot();
+});
+
+test('Callout only description', async () => {
```

Comment:
I'd make a test case "only description and closable"

Perhaps you could rewrite all these tests using a `test.each + table`

Reply:
Ok I'll include that new case, but I think I prefer having mistica form fields than story arg

Reply:
done